required/latex-lab/tagnames-commands.md

```diff
@@ -0,0 +1,98 @@
+# commands for tagnames
+
```

Comment:
Oops: the word you need here is analysed/analyzed(?) —
not at all the same:-).

Reply:
close enough :-), no?

Reply:
well now I know where this file disappeared. I started it some time ago to get an overview over all the command names and then couldn't find it anymore. Yes, xls is fine with me. At the begin I thought it would be a rather simple table, but then more and more variants appeared.

Reply:
I'm going to producean xls in a separate branch